utils/parsers.py

```diff
@@ -480,4 +480,6 @@ def _custom_request_validation(self, path, method, request_data):
 class GHCAPIParser(APIParser):
     """Parser class for the GHC API."""
 
-    api_file = "https://raw.githubusercontent.com/transcom/mymove/master/swagger/ghc.yaml"
+    api_file = (
+        "https://raw.githubusercontent.com/transcom/mymove/0d619037ee393bdba7ea9afa93b5a18529c2d904/swagger/ghc.yaml"
```

Comment:
@duncan-truss is this a code change that'll be reverted when the mymove PR lands?

Reply:
Yes, hopefully I'll remember to switch it back once this mymove PR merges in first